src/hooks/useCds/translate.js

```diff
@@ -171,6 +171,8 @@ function mapLoincCode(result, loincMapping) {
 }
 
 function mapResult(result, loincMapping, testCodeResultMapping) {
+  if (!result.code.coding || !result.code.coding.length > 0) {return};
```

Comment:
if (!result.code.coding?.length) {  return; }

Reply:
Good idea, that is more succinct. I just made the suggested change in the most recent commit: https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/25/commits/541e41ff6f8f70b727f1ed2db3329db3f015f9b2.